docs/OV_Runtime_UG/automatic_batching.md

```diff
@@ -27,7 +30,7 @@ Batching is a straightforward way of leveraging the GPU compute power and saving
 @endsphinxtabset
 
 
-> **NOTE**: You can disable the Auto-Batching (for example, for the GPU device) from being triggered by the `ov::hint::PerformanceMode::THROUGHPUT`. To do that, pass the `ov::hint::allow_auto_batching` set to **false** in addition to the `ov::hint::performance_mode`:
+> **NOTE**: Auto-Batching can be disabled (for example, for the GPU device) to prevent being triggered by `ov::hint::PerformanceMode::THROUGHPUT`. To do that, set `ov::hint::allow_auto_batching` to **false** in addition to the `ov::hint::performance_mode`, as shown below:
```

Comment:
As I commented earlier, this note and the code below belong together. This formatting is problematic. We can put them under a ### Disabling Auto-Batching heading after the next sentence.

Reply:
It's been done.